lms/djangoapps/email_marketing/signals.py

```diff
@@ -7,24 +7,54 @@
 import crum
 from django.conf import settings
 from django.dispatch import receiver
-from sailthru.sailthru_client import SailthruClient
 from sailthru.sailthru_error import SailthruClientError
 from celery.exceptions import TimeoutError
 
 from email_marketing.models import EmailMarketingConfiguration
+from openedx.core.djangoapps.waffle_utils import WaffleSwitchNamespace
 from lms.djangoapps.email_marketing.tasks import update_user, update_user_email, get_email_cookies_via_sailthru
 from openedx.core.djangoapps.lang_pref import LANGUAGE_KEY
 from student.cookies import CREATE_LOGON_COOKIE
+from student.signals import ENROLL_STATUS_CHANGE
 from student.views import REGISTER_USER
 from util.model_utils import USER_FIELD_CHANGED
+from .tasks import update_course_enrollment
 
 log = logging.getLogger(__name__)
 
+AUDIT = 'audit'
+HONOR = 'honor'
+
+# Modes utilized for audit/free enrollments
+AUDIT_MODES = [AUDIT, HONOR]
```

Comment:
Why do you still defined those modes by strings? Why not use the link to the course_mode module I gave you?